suite-native/transactions/src/components/TransactionDetail/TransactionDetailCoinListItem.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import { TouchableOpacity } from 'react-native-gesture-handler';
+
+import { useNavigation } from '@react-navigation/native';
+
+import { Box, Text } from '@suite-native/atoms';
+import { AccountKey, WalletAccountTransaction } from '@suite-common/wallet-types';
+import { EthereumTokenTransfer, CoinIcon } from '@suite-native/ethereum-tokens';
+import {
+    StackNavigationProps,
+    RootStackParamList,
+    RootStackRoutes,
+} from '@suite-native/navigation';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+
+import { TokenTransferListItemValues } from '../TransactionsList/TokenTransferListItem';
+import { TransactionListItemValues } from '../TransactionsList/TransactionListItem';
+import {
+    transactionListItemContainerStyle,
+    valuesContainerStyle,
+} from '../TransactionsList/TransactionListItemContainer';
+
+type TransactionCoinListItemProps = {
+    accountKey: AccountKey;
+    transaction: WalletAccountTransaction;
+    tokenTransfer?: EthereumTokenTransfer;
+    isFirst?: boolean;
+    isLast?: boolean;
+    onPress: () => void;
+};
+
+type TransactionDetailNavigation = StackNavigationProps<
+    RootStackParamList,
+    RootStackRoutes.TransactionDetail
+>;
+
+export const iconContainerStyle = prepareNativeStyle(utils => ({
+    alignItems: 'center',
+    justifyContent: 'center',
+    padding: 12,
+    borderRadius: utils.borders.radii.round,
+    backgroundColor: utils.colors.backgroundSurfaceElevation2,
+    marginRight: utils.spacings.medium,
+}));
+
+const CoinNameContainerStyle = prepareNativeStyle(_ => ({
+    flexShrink: 1,
+}));
+
+export const TransactionDetailCoinListItem = ({
```

Comment:
No need to have `Coin` in name of component

Reply:
renamed 1e27599ac